### DISCUSSION TITLE
Revoked OAuth app still persist in pull request's side panel

### DISCUSSION THREAD

**Discussion by Allirey** [community/community#106706] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

I used to be a member of an organization that added an Oauth app (everhour - a time tracking app) and I authorized it. After a while, **I revoked access** to it in settings/applications on github, and now that I'm not a member of the organization, I'm still seeing this integration ("connect" button) in **all** of my pull requests in all repos.

I checked settings/applications and there is no such app anymore.

Is it broken revoke feature from 3rd party app side or it's github bug? Any possible way I can get rid of this integration?
<details>
  <summary>How it looks for me in each PR</summary>
  
![Screenshot from 2024-02-08 01-37-29](https://github.com/community/community/assets/22729337/9bcb9c89-f744-429f-afb3-663c0ab154b3)
</details>

**Comment by devnavodhimsara** [comment]
This behavior is very likely caused by the Everhour browser extension, not a bug within GitHub or the OAuth revocation process itself.

Everhour relies on a browser extension to inject that specific UI component (the timer/connect button) directly into the web pages of tools like GitHub, Trello, etc.

Even though you successfully revoked the OAuth access on GitHub's end in your settings, the Everhour extension is still installed and active in your web browser. Because the extension detects it is on a GitHub PR page but no longer has the authorized connection, it defaults to showing that "Connect" prompt.

To fix this:
Simply check your browser's installed extensions and either disable or completely remove the Everhour extension. Once you do that and refresh the GitHub page, that side panel integration should disappear immediately from all your PRs.

Hope this helps! Let me know if that solves the issue for you.

**Comment by MahdiJDS** [comment]
Hi! This doesn’t sound like a “user mistake” at all—if the app no longer exists in Settings → Applications, GitHub should stop offering it, so it may be stuck state on GitHub’s side (or cached permissions).

A few things you can try to fully remove it:

Re-check org-level access / policy
Make sure the OAuth app isn’t still authorized at the organization level (even if your membership ended). Sometimes org installs/approvals remain until removed by an org admin.
Look for the integration in your “Authorized GitHub Apps/OAuth apps”
In addition to Settings → Applications, check whether GitHub still lists it under the newer “authorized apps” areas (names vary depending on UI updates).
Clear site data / hard refresh
If you’re still seeing the “Connect” button everywhere, it could be UI cache. Try:
hard refresh / incognito mode
clear cookies/cache for [github.com](http://github.com/)
try another browser
Try disconnect via the third-party app
Sometimes the app still expects the OAuth link on its side. If Everhour has a “Connected GitHub” / “Disconnect” option, revoke there too.
Check if it’s appearing only for PRs from certain branches
Rarely, it’s tied to where the integration was installed or where permissions were granted.
If none of the above works, I’d treat it as a GitHub caching/state bug. The most helpful info for GitHub support would be: your GitHub username, approximate time when you revoked access, and whether the “connect” button appears in incognito / other browsers.

Hope this helps—happy to dig in further if you tell me what browser and whether it shows up in incognito as well.
